feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts

```diff
@@ -97,6 +109,50 @@ export class QuickOrderComponent implements OnInit, OnDestroy {
     this.cartErrors$.next([]);
   }
 
+  clearWarnings(): void {
+    this.cartWarnings$.next([]);
+  }
+
+  clearSuccesses(): void {
+    this.cartSuccesses$.next([]);
+  }
+
+  protected verifyErrors(errors: CartAddEntrySuccessEvent[]): void {
+    const noAddedEntries = errors.filter((error) => error.quantityAdded === 0);
+
+    this.setErrors(noAddedEntries);
+  }
+
+  protected verifyWarnings(errors: CartAddEntrySuccessEvent[]): void {
+    const warnings = errors.filter((error) => error.quantityAdded !== 0);
+
+    this.setWarnings(warnings);
+  }
+
+  protected verifySuccesses(
+    errors: CartAddEntrySuccessEvent[],
+    entries: OrderEntry[]
+  ): void {
+    const successAddedEntries: OrderEntry[] = [];
+
+    entries.forEach((entry) => {
+      const element = errors.find(
+        (error) => error.productCode === entry.product?.code
+      );
+      if (!element) {
+        successAddedEntries.push(entry);
+      }
+    });
+
+    this.setSuccesses(successAddedEntries);
+  }
+
+  protected clearErrorsAndWarnings(): void {
```

Comment:
How about changing name of this function to `clearStatuses`? As it's not only warnings and errors ;)